packages/base-driver/lib/basedriver/driver.ts

```diff
@@ -162,7 +163,12 @@ export class BaseDriver<
     // log timing information about this command
     const endTime = Date.now();
 
-    cmd = this.clarifyCommandName(cmd, args);
```

Comment:
it would still make sense to keep this logic in a separate method just in case we'd need to patch it in descendant classes